engine/admin/logging/awslogs.md

```diff
@@ -63,6 +63,17 @@ specified, the container ID is used as the log stream.
 > at a time.  Using the same log stream for multiple containers concurrently
 > can cause reduced logging performance.
 
+### tag
+
+Specify tag as an alternative to the AWS logstream name, which interpret some markup, ex `{{.ID}}`, `{{.FullID}}` or `{{.Name}}` `docker.{{.ID}}`
```

Comment:
How about: Specify `tag` as an alternative to the `awslogs-stream` option.  `tag` interprets template markup (e.g., `{{.ID}}`, `{{.FullID}}`, `{{.Name}}`, or `docker.{{.ID}}`).  See the [tag option documentation](log_tags.md) for details on all supported template substitutions.  When both `awslogs-stream` and `tag` are specified, the value supplied for `awslogs-stream` will override the template specified with `tag`.